src/index.js

```diff
@@ -1 +1,18 @@
-console.log("all is good");
\ No newline at end of file
+import {loadHome} from './homecontent'
+import {loadContact} from './contact'
+import {loadAbout} from './aboutcontent'
+
+
+window.onload = () => {
```

Comment:
We load our script at the end of the html file so we don't have use `window.onload` here.